src/test/java/com/google/cloud/tools/maven/CloudSdkAppEngineFactoryTest.java

```diff
@@ -94,130 +88,26 @@ public void wireUp()
     File outFolder = tempFolder.newFolder("tempOut");
     when(buildMock.getDirectory()).thenReturn(outFolder.getAbsolutePath());
 
-    when(cloudSdkFactoryMock.cloudSdkBuilder()).thenReturn(cloudSdkBuilderMock);
-    when(cloudSdkBuilderMock.build()).thenReturn(cloudSdkMock);
-
     doReturn(INSTALL_SDK_PATH).when(cloudSdkDownloader).downloadCloudSdk(logMock);
-    doNothing().when(cloudSdkChecker).checkCloudSdk(cloudSdkMock);
-  }
-
-  @Test
-  public void testStandardStaging() throws CloudSdkNotFoundException {
-    // invoke
-    factory.standardStaging();
-
-    // verify
-    verify(cloudSdkBuilderMock).build();
-    verify(cloudSdkFactoryMock).standardStaging(cloudSdkMock);
-    verifyDefaultCloudSdkBuilder();
-  }
-
-  @Test
-  public void testFlexibleStaging() {
-    // invoke
-    factory.flexibleStaging();
-
-    // verify
-    verify(cloudSdkFactoryMock).flexibleStaging();
   }
 
   @Test
-  public void testDeployment() throws CloudSdkNotFoundException {
-    // invoke
-    factory.deployment();
-
-    // verify
-    verify(cloudSdkBuilderMock).build();
-    verify(cloudSdkFactoryMock).deployment(cloudSdkMock);
-    verifyDefaultCloudSdkBuilder();
+  public void testGetGcloud() {
+    factory.getGcloud();
+    verify(mojoMock).getArtifactId();
+    verify(mojoMock).getArtifactVersion();
+    verify(mojoMock).getServiceAccountKeyFile();
+    verifyNoMoreInteractions(mojoMock);
   }
 
   @Test
-  public void testDevServer1RunSync() throws CloudSdkNotFoundException {
-    // invoke
-    factory.devServerRunSync(SupportedDevServerVersion.V1);
-
-    // verify
-    verify(cloudSdkBuilderMock).build();
-    verify(cloudSdkFactoryMock).devServer1(cloudSdkMock);
-    verifyDefaultCloudSdkBuilder();
-  }
-
-  @Test
-  public void testDevServer2RunSync() throws CloudSdkNotFoundException {
```

Comment:
Are these run/stop tests not relevant anymore?

Reply:
not really, they never did anything, and the underlying mechanism is different now.